packages/ef-runtime-client/index.js

```diff
@@ -0,0 +1,39 @@
+const EFRegistry = {};
+
+export function init(overrides = {}) {
+  // Add SystemJS script tag
+  const systemJSScript = document.createElement("script");
+  systemJSScript.src =
+    "https://cdnjs.cloudflare.com/ajax/libs/systemjs/6.14.2/system.min.js";
```

Comment:
We should bundle this into the published version of this package, instead of relying on a separate CDN distribution, that way we benefit from H2 loading on FT.com.